tools/analysis_tools/analyze_logs.py

```diff
@@ -174,17 +151,28 @@ def load_json_logs(json_logs):
     for json_log, log_dict in zip(json_logs, log_dicts):
         with open(json_log, 'r') as log_file:
             epoch = 1
-            for line in log_file:
+            for i, line in enumerate(log_file):
                 log = json.loads(line.strip())
+                val_flag = False
                 # skip lines only contains one key
                 if not len(log) > 1:
                     continue
+
                 if epoch not in log_dict:
                     log_dict[epoch] = defaultdict(list)
+
                 for k, v in log.items():
-                    log_dict[epoch][k].append(v)
+                    if '/' in k:
+                        log_dict[epoch][k.split('/')[-1]].append(v)
+                        val_flag = True
+                    elif val_flag:
```

Comment:
Are you sure about this line of modification? We'll appreciate it if you can run these `analyze_logs` scripts to make it bug-free.

Reply:
This script is just modified from [mmdet](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/tools/analysis_tools/analyze_logs.py). I will check it ASAP.